src/api/ledger/parse/trustline.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const assert = require('assert');
+const utils = require('./utils');
+const flags = utils.core.Transaction.flags.TrustSet;
+
+function parseTrustline(tx) {
```

Comment:
If we change this, we would also have to change `parseOrder` -> `parseOfferCreate` and then it starts getting confusing with questions like "what's the difference between an order and an offer?" I think it's better to just stick with consistent naming.